pallets/ah-migrator/src/types.rs

```diff
@@ -25,3 +25,17 @@ pub fn map_lock_reason(reasons: LockReasons) -> LockWithdrawReasons {
 		LockReasons::Misc => LockWithdrawReasons::TIP,
 	}
 }
+
+/// Relay Chain pallet list with indexes.
+#[derive(Encode, Decode)]
+pub enum RcPalletConfig {
+	#[codec(index = 255)]
+	RcmController(RcMigratorCall),
+}
+
+/// Call encoding for the calls needed from the rc-migrator pallet.
+#[derive(Encode, Decode)]
+pub enum RcMigratorCall {
+	#[codec(index = 2)]
+	StartDataMigration,
```

Comment:
I think this should maybe be something like `AhStageChanged(stage)` and then define the stage type either in the RC pallet or in a shared crate.

Reply:
May be `receive_stage_update(stage)`? If we have many, it makes total sense. If we have three, may be three different calls is simpler solution. But no strong opinion. Again I would may be wait until we see the results of dry run.